SELearning.Core/Content/Section.cs

```diff
@@ -10,4 +10,7 @@ public class Section
     {
         return null;
     }
+
+    public bool Equals(Section other)
```

Comment:
Why isn't it sufficient to check the id's? Can there ever exist two sections with identical id?

Reply:
The equals method shouldn't really care about the semantics in our program - it just checks if two instances have the same fields (i.e. they are equal objects).

Reply:
True